src/Actions/VatRate/UpdateVatRate.php

```diff
@@ -31,4 +31,16 @@ public function performAction(): Model
 
         return $vatRate->withoutRelations()->fresh();
     }
+
+    protected function prepareForValidation(): void
+    {
+        if (($this->data['is_default'] ?? false)
+            && ! resolve_static(VatRate::class, 'query')
+                ->whereKeyNot($this->data['id'] ?? 0)
+                ->where('is_default', true)
+                ->exists()
+        ) {
+            $this->rules['is_default'] .= '|accepted';
```

Comment:
could be an array when getting overridden

Reply:
could be string, array, or maybe doesn't even exist. This is a mare general issue and needs to be addressed in a different pull request.